src/app/config.js

```diff
@@ -0,0 +1,222 @@
+/* jshint maxlen:false */
+define([
+    'dojo/has',
+    'dojo/request/xhr',
+
+    'esri/config'
+], function (
+    has,
+    xhr,
+
+    esriConfig
+) {
+    // force api to use CORS on mapserv thus removing the test request on app load
+    // e.g. http://mapserv.utah.gov/ArcGIS/rest/info?f=json
+    esriConfig.defaults.io.corsEnabledServers.push('mapserv.utah.gov');
+    esriConfig.defaults.io.corsEnabledServers.push('basemaps.utah.gov');
+    esriConfig.defaults.io.corsEnabledServers.push('api.mapserv.utah.gov');
+    // esriConfig.defaults.io.corsEnabledServers.push('192.168.230.138');
+
+    window.AGRC = {
+        // errorLogger: ijit.modules.ErrorLogger
+        errorLogger: null,
+
+        // app: app.App
+        //      global reference to App
+        app: null,
+
+        // version.: String
+        //      The version number.
+        version: '1.0.0',
+
+        // apiKey: String
+        //      The api key used for services on api.mapserv.utah.gov
+        apiKey: '', // acquire at developer.mapserv.utah.gov
+
+        // exportWebMapUrl: String
+        //      print task url
+        exportWebMapUrl: 'http://mapserv.utah.gov/arcgis/rest/services/Utilities/PrintingTools/GPServer/Export%20Web%20Map%20Task',
+
+        urls: {
+            facilities: 'http://192.168.230.147/arcgis/rest/services/HealthFacalities/MapServer/0'
+        },
+
+        fieldNames: {
+            facType: 'TYPE', // Health facilities type
+            facSubtype: 'FACTYPE' // Health facilities subtype
+        },
+        featureClassNames: {
+            counties: 'SGID10.Boundaries.Counties',
+            municipalities: 'SGID10.Boundaries.Municipalities',
+            landOwnership: 'SGID10.Cadastre.LandOwnership',
+            nationalGrid: 'SGID10.Indices.NationalGrid'
+        },
+
+        filters: [
```

Comment:
the indenting on this section of code is pretty bad. ESLint can fix that for you.